flow-typed/publish.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
- [ ] Running uploads sometimes lost on refresh
    - Repro:  Had 3 items, with the bottom 2 lost after a refresh. Both items have the same filename,
        - Stopped
        - Uploading 4%
        - Uploading 1%